streaming/base/dataset.py

```diff
@@ -311,10 +321,16 @@ def __init__(self,
                 f'Invalid sampling method: {sampling_method}. Must be one of `balanced` or `fixed`.'
             )
 
+        # Check batching method is one of "random", "stratified", or "per_stream".
+        if self.batching_method not in ['random', 'stratified', 'per_stream']:
+            raise ValueError(
+                f'Invalid batching method: {batching_method}. Must be one of `random`, `stratified`, or `per_stream.'
+            )
+
         # issue deprecation warning for py1b shuffle algorithm.
         if self.shuffle_algo == 'py1b':
-            warn(
-                'The \'py1b\' shuffle algorithm will soon be deprecated. Please use the more performant \'py1br\' algorithm instead.',
+            warnings.warn(
+                'The \'py1b\' shuffle algorithm will soon be deprecated. Please use the more performant \'py1e\' or \'py1br\' algorithms instead.',
```

Comment:
Wondering, should we stick to recommending just `py1br` since it is a better alternative of `py1b` because `py1e` shuffling is great but not at par with `py1br` ??

Reply:
the shuffle quality is just a bit lower, imo in most cases py1e > py1br > py1b?

Reply:
fixed!